### ISSUE TITLE
2023-08-13 dvcapture not capturing

### ISSUE THREAD

**Issue by libbyhopfauf** [mipops/dvrescue#724]
With the latest build of dvrescue, the GUI is not capturing or successfully remote controlling decks. I've tested this on a Sony DSR-25 and a DSR-1500A and both are recognized as connected, but neither seem to be communicating with dvrescue (will not play, FF, REW, etc via the deck control buttons).

**Comment by JeromeMartinez** [comment]
@libbyhopfauf what is the latest working version you used?

**Comment by tomackroyd** [comment]
I have been trying to install this GUI too. After installing the current version the capture window was showing just a blank white screen. I then discovered this GitHub issue and installed the 2023-07-15 versions. The capture window now says "NO DV DECK DETECTED". The vrecord GUI detects the deck and can successfully digitise, as can iMovie and QT Player.
I have run through all the recommended steps at https://mipops.github.io/dvrescue/sections/installation.html
MacBook Air M2, macOS Venture 13.4.1 ©
DV Rescue GUI version: devel Jul 15 2023
dvrescue CLI version: 0.22.10
Interestingly "vrecord -e" GUI deck control is flaky, sometimes works, sometimes not.
 Can supply other details if asked.

**Comment by tomackroyd** [comment]
I have just installed dvrescue 0.21.11.20231025
This version eventually detects the deck but shows "parsing error" and "NO SIGNAL" notes and does not show pictures from tape. Deck controls work but with a three second or so lag. Two attempted captures resulted in empty dv files.
[CHINA_TAPE_5_DVRESCUE_TEST_2.dv.capture.log](https://github.com/mipops/dvrescue/files/13194182/CHINA_TAPE_5_DVRESCUE_TEST_2.dv.capture.log)
[CHINA_TAPE_5_DVRESCUE_TEST_2.dv.dvrescue.xml.txt](https://github.com/mipops/dvrescue/files/13194183/CHINA_TAPE_5_DVRESCUE_TEST_2.dv.dvrescue.xml.txt)
<img width="1470" alt="CHINA_TAPE_5_DVRESCUE_TEST_SCREENSHOT_1" src="https://github.com/mipops/dvrescue/assets/58644101/a6197ee9-1ba6-4284-aba3-c0246236cf54">
<img width="1470" alt="CHINA_TAPE_5_DVRESCUE_TEST_SCREENSHOT_2" src="https://github.com/mipops/dvrescue/assets/58644101/a93fd20b-fed4-43ba-aaf3-d34aad207fff">
<img width="1470" alt="CHINA_TAPE_5_DVRESCUE_TEST_SCREENSHOT_3" src="https://github.com/mipops/dvrescue/assets/58644101/ec97cef9-d771-4008-998b-850171a96381">
<img width="1470" alt="CHINA_TAPE_5_DVRESCUE_TEST_SCREENSHOT_4" src="https://github.com/mipops/dvrescue/assets/58644101/cf3acbcb-268c-4e54-aafd-3ee6d5ab66ab">

Media Info for resulting .dv file:
CHINA_TAPE_5_DVRESCUE_TEST_2.dv
File size: 0.00 Byte

**Comment by dericed** [comment]
Regarding the parsing error, can you send 'dvrescue -list-devices-json'.

**Comment by tomackroyd** [comment]
List devices returns "DV":
twelveplusone@Toms-MacBook-Air ~ % dvrescue --list_devices -json
DV
twelveplusone@Toms-MacBook-Air ~ %

**Comment by dericed** [comment]
Can you remove the space before -json and rerun.

**Comment by tomackroyd** [comment]
Error: missing value after --list_devices-json.

**Comment by g-maxime** [comment]
Is --list_devices_json

**Comment by tomackroyd** [comment]
Does this help? avfctl -list_devices
2023-10-29 09:07:01.911 avfctl[55154:2651497] Devices:
2023-10-29 09:07:01.911 avfctl[55154:2651497] [0] DV

**Comment by dericed** [comment]
sorry, my bad for trying type this on my phone `dvresuce --list_devices_json`

**Comment by dericed** [comment]
`[0] DV`? And that's a Sony DSR-40P?

**Comment by tomackroyd** [comment]
@dericed Yes
dvrescue --list_devices_json returns:
Error: missing value after --list_devices_json.

**Comment by dericed** [comment]
> Does this help? avfctl -list_devices 2023-10-29 09:07:01.911 avfctl[55154:2651497] Devices: 2023-10-29 09:07:01.911 avfctl[55154:2651497] [0] DV

Helps a bit, but the dvrescue gui is parsing the json output of `dvresuce --list_devices_json` so that helps more.

**Comment by dericed** [comment]
@tomackroyd can you update to https://old.mediaarea.net/download/snapshots/binary/dvrescue/20231028/ and retry

**Comment by tomackroyd** [comment]
@dericed Done. Same result.
[CHINA_TAPE_5_DVRESCUE_TEST_3.dv.dvrescue xml.txt](https://github.com/mipops/dvrescue/files/13196826/CHINA_TAPE_5_DVRESCUE_TEST_3.dv.dvrescue.xml.txt)
[CHINA_TAPE_5_DVRESCUE_TEST_3.dv.capture.log](https://github.com/mipops/dvrescue/files/13196827/CHINA_TAPE_5_DVRESCUE_TEST_3.dv.capture.log)
[DVRESCUE_SESSIONS_2023-10-29_COMMAND_LOG_1.log](https://github.com/mipops/dvrescue/files/13196828/DVRESCUE_SESSIONS_2023-10-29_COMMAND_LOG_1.log)

**Comment by tomackroyd** [comment]
> Is --list_devices_json

```
~ % Is --list_devices_json
zsh: command not found: Is
```

**Comment by dericed** [comment]
Is was a verb. Try that option with dvrescue.

**Comment by tomackroyd** [comment]
```
~ % dvrescue --list_devices_json
Error: missing value after --list_devices_json.
```

**Comment by tomackroyd** [comment]
Interesting that dvrescue_GUI_0.21.11.20231006_Mac.app sees the deck, but instead of the parsing error, identifies it (incorrectly) as "[0] DV-VCR (Sony GV-D1000)" also has a "NO SIGNAL" screen.
dvrescue_GUI_0.21.11.20231025_Mac.app still has the ***parsing error***

**Comment by dericed** [comment]
Hi @tomackroyd, it still seems that you don't have a recent dvrescue cli installed.

for me, I get:
```
$ dvrescue --version
DVRescue v.0.21.11.20231030 (MediaInfoLib v.23.10) by MIPoPS.


$ dvrescue --list_devices_json
[{"id":"0x800460104f1243e","name":"DV-VCR (Sony HVR-M15U)","type":"DV"}]
```

Could you run:

```
which dvrescue ; ls -l $(which dvrescue) ; dvrescue --version
```

Or perhaps you have multiple dvrescue cli in your path.

**Comment by tomackroyd** [comment]
```
twelveplusone@Toms-MacBook-Air ~ % brew upgrade dvrescue
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> New Formulae
daktilo                    lager                      modsurfer                  pmix                       sigma-cli                  zug
==> New Casks
deveco-studio                                         metamer                                               music-decoy

You have 8 outdated formulae installed.

Warning: mediaarea/mediaarea/dvrescue 0.22.10 already installed
twelveplusone@Toms-MacBook-Air ~ % which dvrescue ; ls -l $(which dvrescue) ; dvrescue --version
/opt/homebrew/bin/dvrescue
lrwxr-xr-x  1 twelveplusone  admin  39 26 Oct 18:46 /opt/homebrew/bin/dvrescue -> ../Cellar/dvrescue/0.22.10/bin/dvrescue
DVRescue v.0.22.10 (MediaInfoLib v.22.09) by MIPoPS.
twelveplusone@Toms-MacBook-Air ~ % dvrescue --version 
DVRescue v.0.22.10 (MediaInfoLib v.22.09) by MIPoPS.
twelveplusone@Toms-MacBook-Air ~ % 
```

**Comment by dericed** [comment]
yup, so all my earlier advise applies to `/usr/local/bin/dvrescue --version`. You could also run `brew unlink dvrescue` to kick the old release out of the way.

**Comment by tomackroyd** [comment]
Thanks for the fix. Not sure how I ended up with two versions - I normally use home-brew and followed the instructions at https://mipops.github.io/dvrescue/sections/installation.html - but the standalone CLI installers put it in a different place - is this something that will trip other users?

**Comment by libbyhopfauf** [comment]
@tomackroyd Yes, that is an outdated part of the instructions. I will update it now! Thank you for letting us know.

**Comment by tomackroyd** [comment]
I'm still unsure how to continue using dvrescue without using Homebrew. Could anyone do me an explainer?
```
Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  dvrescue
```

**Comment by libbyhopfauf** [comment]
> I'm still unsure how to continue using dvrescue without using Homebrew. Could anyone do me an explainer? @dericed I followed your advice and unlinked dvrescue, but of course now I get this when running `brew doctor`
> 
> ```
> Warning: You have unlinked kegs in your Cellar.
> Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
> those kegs to fail to run properly once built. Run `brew link` on these:
>   dvrescue
> ```

@tomackroyd would you mind running the following commands to update homebrew:
```
brew update
brew upgrade
brew cleanup
brew doctor
```

And then install the [2023-11-11 version of the DVRescue GUI](https://mediaarea.net/download/snapshots/binary/dvrescue-gui/20231111/) and open it directly from the install window?

I was experiencing similar issues as you were with previous versions, but they now seem to be resolved with the latest version, so am curious to know if it works for you now.

**Comment by tomackroyd** [comment]
@libbyhopfauf I ran all those commands, downloaded (actually the latest) dvrescue_GUI_0.21.11.20231113_Mac.dmg, and launched it directly from the disk image Finder window.

```
twelveplusone@Toms-MacBook-Air ~ % /opt/homebrew/Cellar/dvrescue/0.22.10/bin/dvrescue --version
DVRescue v.0.22.10 (MediaInfoLib v.22.09) by MIPoPS.
twelveplusone@Toms-MacBook-Air ~ % /usr/local/bin/dvrescue --version  
DVRescue v.0.21.11.20231106 (MediaInfoLib v.23.10) by MIPoPS.
```
```
twelveplusone@Toms-MacBook-Air ~ % dvrescue --list_devices_json          
[{"id":"0x800460101ab28fb","name":"DV","type":"DV"}]

```
dvrescue GUI identifies the deck as DV (DV), transport control works, still "NO SIGNAL"

I attempted a capture, resulting in an empty file
[test.dv.capture.log](https://github.com/mipops/dvrescue/files/13383125/test.dv.capture.log)
[test.dv.dvrescue_xml.txt](https://github.com/mipops/dvrescue/files/13383128/test.dv.dvrescue_xml.txt)

**Comment by tomackroyd** [comment]
I should add that vrecord is still not working for me, pending me being able to test https://github.com/amiaopensource/vrecord/pull/770 once vrecord is updated.

**Comment by libbyhopfauf** [comment]
@tomackroyd would you mind trying the latest builds of vrecord and [dvrescue](https://mediaarea.net/download/snapshots/binary/dvrescue-gui/20231208/) and let us know if you're able to capture now?

Also, the no signal just means that the device is connect successfully to the DVRescue GUI, it just isn't playing/the software isn't receiving any information from the device. Just to double check, is your deck in remote mode? That might be why it's failing to play and the recording is just an empty file.

**Comment by tomackroyd** [comment]
@libbyhopfauf will do, just may be a day or two. DVRescue does play the tape and captures info, just not picture and sound! See logs
Tom

**Comment by tomackroyd** [comment]
@libbyhopfauf How do I install the latest vrecord build? Homebrew is saying latest version is 2023-08-07

**Comment by tomackroyd** [comment]
Have tested with dvrescue_GUI_22.12.20231208_Mac - no change:

- DVRescue recognises device as DV (DV)
- No signal
- Transport controls work only sometimes
- Test capture creates empty file

DVRescue generates following logs from capture process:
`FramePos,abst,abst_r,abst_nc,tc,tc_r,tc_nc,rdt,rdt_r,rdt_nc,rec_start,rec_end,Used,Status,Comments,BlockErrors,BlockErrors_Even,IssueFixed,SourceSpeed,FrameSpeed,InputPos,OutputPos
`
```
<?xml version="1.0" encoding="UTF-8"?>
<dvrescue xmlns="https://mediaarea.net/dvrescue" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://mediaarea.net/dvrescue https://mediaarea.net/dvrescue/dvrescue.xsd" version="1.2.1">
	<creator>
		<program>dvrescue</program>
		<version>0.21.11.20231025</version>
		<library version="23.10">MediaInfoLib</library>
	</creator>
	<media ref="-" fromCapture="device://0x800460101ab28fb" format="DV">
		<frames count="175" pts="00:00:00.000000" end_pts="00:00:07.000000" size="720x576" video_rate="25" chroma_subsampling="4:2:0" aspect_ratio="4/3" audio_rate="48000" channels="2" speed="33">
			<frame n="0" pos="0" pts="00:00:00.000000" abst="7847" tc="00:00:26:04" rdt="2001-03-08 17:33:31" seqn="4"/>
			<frame n="174" pos="25056000" pts="00:00:06.960000" abst="9935" tc="00:00:33:03" rdt="2001-03-08 17:33:38" seqn="A"/>
		</frames>
	</media>
</dvrescue>
```
Other terminal probes:
```
twelveplusone@Toms-MacBook-Air ~ % /usr/local/bin/dvrescue --version
DVRescue v.0.21.11.20231106 (MediaInfoLib v.23.10) by MIPoPS.
```
```
twelveplusone@Toms-MacBook-Air ~ % avfctl -list_devices                            
2023-12-12 20:41:17.255 avfctl[25594:857950] Devices:
2023-12-12 20:41:17.257 avfctl[25594:857950] [0 - 0x800460101ab28fb] DV
twelveplusone@Toms-MacBook-Air ~ % dvrescue --list_devices_json
[{"id":"0x800460101ab28fb","name":"DV","type":"DV"}]
```

vrecord still cannot see deck

Tom

**Comment by libbyhopfauf** [comment]
@tomackroyd just to double check, is your deck or camcorder in remote mode? Also, have you checked/changed your security policy recently to allow dvrescue to capture? Here are the [instructions for changing your permissions](https://mipops.github.io/dvrescue/sections/installation.html) so that macOS allows you to record using dvrescue (jump to the "troubleshooting" section). I had to reset mine recently to get dvcapture to work for vrecord.

**Comment by 900video** [comment]
Having the same issue (No Signal) and would absolutely love to help be a guinea pig to help resolve it.
- My DSR-40 deck shows up in Vrecord and DVRescue. 
- I have given system prefs full access 
- Have BM Desktop Video 12.7.1 installed
- DVRescue GUI (Jan 17 Version). When I manually push play on my deck, DVRescue can recognize that it's playing as it goes from "Stopped" to "Playing" but there's no signal in the window. 
- I can capture through Quicktime. 
- Is there anything I can do to help test this?  Thanks Jared

**Comment by libbyhopfauf** [comment]
@900video I just replied to your post in issue #808, but I am wondering if you would mind testing the [2024-01-20 version of the daily build](https://mediaarea.net/download/snapshots/binary/dvrescue-gui/20240120/) for the dvrescue GUI? I'm also curious if you have changed your security policy in recovery mode to "reduced security" and checked "allow" for the first option under that setting. 

After adding the software to the applications folder, open the app directly from the install window by double clicking on the DVRescue icon (this will avoid any issues with the GUI using an older version of the CLI that you might have installed on your computer). Also, please wait about 10 seconds for the image to appear in the GUI playback window when recording. The file will contain all of the information, there is just a slight delay currently for the playback display.

I'm just curious because I was able to capture in the DVRescue GUI using the version above. Test files and logs are located [here](https://drive.google.com/drive/folders/1ZDLIqVl1RQHqWAsnfHjwvk5MzLE2jG6h?usp=drive_link).

Here are the specs that I am working with:

- computer = Macbook Pro (M2 Max Chip)
- macOS = Sonoma 14.2.1 (23C71) 
- device = Sony DSR-45
- dvrescue version = 2024-01-20
- security policy = reduced
- full disk access = granted for dvrescue and Terminal

**Comment by libbyhopfauf** [comment]
> @900video I just replied to your post in issue #808, but I am wondering if you would mind testing the [2024-01-20 version of the daily build](https://mediaarea.net/download/snapshots/binary/dvrescue-gui/20240120/) for the dvrescue GUI? I'm also curious if you have changed your security policy in recovery mode to "reduced security" and checked "allow" for the first option under that setting.
> 
> After adding the software to the applications folder, open the app directly from the install window by double clicking on the DVRescue icon (this will avoid any issues with the GUI using an older version of the CLI that you might have installed on your computer). Also, please wait about 10 seconds for the image to appear in the GUI playback window when recording. The file will contain all of the information, there is just a slight delay currently for the playback display.
> 
> I'm just curious because I was able to capture in the DVRescue GUI using the version above. Test files and logs are located [here](https://drive.google.com/drive/folders/1ZDLIqVl1RQHqWAsnfHjwvk5MzLE2jG6h?usp=drive_link).
> 
> Here are the specs that I am working with:
> 
> * computer = Macbook Pro (M2 Max Chip)
> * macOS = Sonoma 14.2.1 (23C71)
> * device = Sony DSR-45
> * dvrescue version = 2024-01-20
> * security policy = reduced
> * full disk access = granted for dvrescue and Terminal

@tomackroyd would you mind testing the latest version as well and confirming your security policy? Thank you so much for all your tests you've been doing!

**Comment by tomackroyd** [comment]
Hi Libby. Have tried everything, no die I'm afraid.
Ran:
brew update
brew upgrade
brew cleanup
brew doctor

computer = Macbook Air (M1)
macOS = Sonoma 14.2.1 (23C71)
device = Sony DSR-40P
device remote = on
dvrescue version = 2024-01-20
security policy = reduced with "allow user management" etc checked
full disk access = granted for dvrescue and Terminal
screen and system audio recording = granted for dvrescue

Launched from dvrescue_GUI_22.12.20240127_Mac.dmg:
/Volumes/dvrescue/dvrescue.app
Deck recognised, waited 10secs, transport works, still NO SIGNAL
Tried with remote off, still NO SIGNAL

Tom

**Comment by tomackroyd** [comment]
Freakishly, I captured this screenshot: QuickTime seeing pictures, DVRescue seeing a different frozen picture but displaying DV metadata:
<img width="1470" alt="Screenshot 2024-01-29 at 12 21 27 PM" src="https://github.com/mipops/dvrescue/assets/58644101/498442eb-d81d-475a-84d4-4da39877fceb">

**Comment by tomackroyd** [comment]
And now, DVrescue seeing moving images at same time as QT, with a delay, cannot replicate, also could not capture using DVrescue:
<img width="1470" alt="Screenshot 2024-01-29 at 12 29 32 PM" src="https://github.com/mipops/dvrescue/assets/58644101/c5e1608b-6f82-40f1-bdf0-d45d5ab9b3cf">

**Comment by tomackroyd** [comment]
I have discovered you *can* add dvrescue to Camera permissions. While there are no + or - buttons in that pane, if you drag the app onto it, it will appear in the list.
Having said that, dvrescue has Camera, Full Disk and Screen Recording permissions, but still NO SIGNAL, even after several restarts and permission resets, and waiting 10 seconds, and with remote on. Occasionally the transport controls stop working and this can only be fixed with a restart. I thinks it's advisable *not* to run DVRescue concurrently with QuickTime Player New Movie Recording!
<img width="827" alt="Screenshot 2024-01-29 at 1 12 51 PM" src="https://github.com/mipops/dvrescue/assets/58644101/d38b6ff8-6081-446a-9075-b376c244b82d">

**Comment by libbyhopfauf** [comment]
@tomackroyd Thank you so much for all this! I will update the documentation to reflect the drag and drop of dvrescue into the camera permissions. 

That makes sense that running dvrescue and QT at the same time would cause a crash from both programs trying to communicate with the deck. I know this sounds like a silly question, but QT hasn't been running every time that you attempt to use the DVRescue GUI, right? Just wanted to check because it looks like you have the same set-up that I do, so I'm not sure why it's not working. 

One other question: the tape that you're trying to run isn't HDV is it? Again, just want to rule out any other possibilities!

**Comment by tomackroyd** [comment]
@libbyhopfauf No, I never have QTP running concurrently, and these are standard def PAL miniDV tapes.

**Comment by libbyhopfauf** [comment]
@tomackroyd okay, I didn't think so (and I really appreciated the visual comparison), just wanted to double-check! And also glad we can rule out HDV. 

I wasn't able to replicate the Camera permissions drag/drop (I think because it's already in there so I can toggle the slider on and off and if I toggle it off and then drag the app in, the switch goes back to on, so that's cool). I added it to the documentation in case some other users are able to. It's great to know there's another option!

I'm so sorry to ask again, but you would mind trying the [latest daily build ](https://mediaarea.net/download/snapshots/binary/dvrescue-gui/20240129/)and let me know if there's any change?

**Comment by libbyhopfauf** [comment]
@tomackroyd @900video would you both mind downloading the [latest release of DVRescue](https://mediaarea.net/DVRescue) (released yesterday) and see if you can capture from the GUI? I will note that I've found the deck control to still be a little buggy for FF/REW, but capture is consistently working on multiple computers at the MIPoPS office and I'm just curious if the release fixes the issues that you two were having. Thank you so much!

**Comment by tomackroyd** [comment]
@libbyhopfauf Running from  the disk image at that link, dvrescue_GUI_24.01_Mac.dmg, DVRescue sees pictures from deck.
- had to relaunch, restart, reopen a few times
- transport controls work less than half the time
- cannot seem to capture anything other than empty files
- DVRescue occasionally shows freeze frames when tape is running
- have to restart/reboot to get away from NO SIGNAL screen
- very difficult to replicate the above!
Tom

**Comment by tomackroyd** [comment]
Have replaced above screen recording - hopefully this will demo some of the issues I have.
Note when DVRescue transport controls are working, sometimes the video display shows a freeze frame when deck is playing. After playing with the controls some more, DVRescue "resets" and scans for decks. 
This results in moving pictures again.
Timecode etc metadata display works when in playback, but when attempting a capture, display shows moving pictures but no time metadata. Results in empty files. Also this happens regardless of whether deck is in remote.
https://www.dropbox.com/scl/fi/j8j0shvjnjrz49pamlb7x/DVRESCUE_TESTS_2024-02-03.mov?rlkey=1f729xkngnk8kvusregbi56aw&dl=0

**Comment by dericed** [comment]
Hi @tomackroyd, I found this last dropbox link helpful and glad to see that your machine got past the No Signal image. Can you retest with the latest daily build at https://old.mediaarea.net/download/snapshots/binary/dvrescue-gui/ and let me know if the playback controls still become non-responsive after the first use of 'stop'.

Once you install the dvrescue daily build in your Applications folder, I suggest to run it via Terminal with `/Applications/dvrescue.app/Contents/MacOS/dvrescue --log` then use the GUI to replicate the issue and then you can grab the log via `cat ~/Library/Containers/net.MediaArea.dvrescue.mac-gui/Data/dvrescue-gui_logs/dvrescue-gui.log`

**Comment by tomackroyd** [comment]
I installed from dvrescue_GUI_22.12.20240425 and am back to "NO DV DECK DETECTED"
Same result following "RESET" within the GUI
Same result following brew upgrade etc
Privacy and Security settings checked.
QuickTime Player sees deck and signal.
Same result following several restarts.

`twelveplusone@Toms-MacBook-Air ~ % /Applications/dvrescue.app/Contents/Helpers/dvrescue --version
DVRescue v.22.12.20240425 (MediaInfoLib v.24.04) by MIPoPS.
twelveplusone@Toms-MacBook-Air ~ % /Applications/dvrescue.app/Contents/Helpers/dvrescue --list_devices_json
[{"id":"0x800460101ab28fb","name":"DV","type":"DV"}]
twelveplusone@Toms-MacBook-Air ~ % dvrescue --version
DVRescue v.24.04 (MediaInfoLib v.24.03) by MIPoPS.
twelveplusone@Toms-MacBook-Air ~ % dvrescue --list_devices_json
[{"id":"0x800460101ab28fb","name":"DV","type":"DV"}]
twelveplusone@Toms-MacBook-Air ~ % `
Log here: https://gist.github.com/tomackroyd/cadf2b9aef8d63d222afbd78aa188d8b)

**Comment by tomackroyd** [comment]
Further to the above, any tips on whether this is a problem:
```
twelveplusone@Toms-MacBook-Air ~ % /usr/local/bin/dvrescue --version
DVRescue v.0.21.11.20231106 (MediaInfoLib v.23.10) by MIPoPS.
twelveplusone@Toms-MacBook-Air ~ % dvrescue --version
DVRescue v.24.04 (MediaInfoLib v.24.03) by MIPoPS.
```

**Comment by libbyhopfauf** [comment]
@tomackroyd thank you for all the testing you have done on DVRescue!! I'm trying to sort out as many issues as possible as we wind down the grant period for the project and wanted to check in and see where you were at with using the tools :)

**Comment by tomackroyd** [comment]
@libbyhopfauf Will fire up the gear and let you know!

**Comment by tomackroyd** [comment]
- Installed dvrescue_GUI_24.07.20241214_Mac
- "NO DV DECK DETECTED"
- QuickTime Player sees deck, can record.
- Here's the log:
https://gist.github.com/tomackroyd/01691129baf499b6366f5a6c09ae53fd

list-devices:
```
twelveplusone@Toms-MacBook-Air ~ % /Applications/dvrescue.app/Contents/Helpers/dvrescue -list_devices_json                              
2024-12-18 18:13:42.691 dvrescue[1936:4163947] WARNING: AVCaptureDeviceTypeExternal is deprecated for Continuity Cameras. Please use AVCaptureDeviceTypeContinuityCamera and add NSCameraUseContinuityCameraDeviceType to your Info.plist.
[{"id":"0x800460101ab28fb","name":"DV","type":"DV"}]
```

**Comment by tomackroyd** [comment]
```
twelveplusone@Toms-MacBook-Air ~ % /Applications/dvrescue.app/Contents/Helpers/dvrescue --version
DVRescue v.22.12.20240629 (MediaInfoLib v.24.06) by MIPoPS.
twelveplusone@Toms-MacBook-Air ~ % dvrescue --version
DVRescue v.24.07 (MediaInfoLib v.24.06) by MIPoPS.
```

**Comment by tomackroyd** [comment]
Just going back through this thread trying to resolve this.

I just re-ran `brew unlink dvrescue`
Results:
```
twelveplusone@Toms-MacBook-Air ~ % /Applications/dvrescue.app/Contents/Helpers/dvrescue --version          
DVRescue v.22.12.20240629 (MediaInfoLib v.24.06) by MIPoPS.
twelveplusone@Toms-MacBook-Air ~ % /Applications/dvrescue.app/Contents/Helpers/dvrescue --list_devices_json
[{"id":"0x800460101ab28fb","name":"DV","type":"DV"}]
twelveplusone@Toms-MacBook-Air ~ % dvrescue --version                                        
DVRescue v.0.21.11.20231106 (MediaInfoLib v.23.10) by MIPoPS.
twelveplusone@Toms-MacBook-Air ~ % dvrescue --list_devices_json                              
[{"id":"0x800460101ab28fb","name":"DV","type":"DV"}]
```
Still "NO DV DECK DETECTED"
Restarted computer, same result.

**Comment by tomackroyd** [comment]
- Created a new APFS container on my M2 MacBook Air.
- Fresh install of Sequoia 15.2.
- Installed DVRescue from dvrescue_GUI_24.07.20241221_Mac.dmg.
- Did not install DVRescue CLI.
- Nothing else installed (eg in Homebrew)
- Followed all instructions at https://mipops.github.io/dvrescue/sections/installation.html.
- Checked QuickTime Player could see deck - yes.
- Still "NO DV DECK DETECTED".

Log here:
https://gist.github.com/tomackroyd/5417a14c9d13a1076d1e7271e161c7ad

Note DVRescue did not ask to access camera. Have been unable to manually grant DVRescue camera access in System Settings/Privacy and Security/Camera

```
tomackroyd@Toms-MacBook-Air ~ % /Applications/dvrescue.app/Contents/Helpers/dvrescue --version
DVRescue v.24.07.20241221 (MediaInfoLib v.24.12) by MIPoPS.
tomackroyd@Toms-MacBook-Air ~ % /Applications/dvrescue.app/Contents/Helpers/dvrescue --list_devices_json
[{"id":"0x800460101ab28fb","name":"DV","type":"DV"}]
```
